core/lib/multivm/src/versions/testonly/mod.rs

```diff
@@ -145,7 +144,7 @@ pub(super) fn default_system_env() -> SystemEnv {
 }
 
 pub(super) fn default_l1_batch(number: L1BatchNumber) -> L1BatchEnv {
-    let timestamp = unix_timestamp_ms();
+    let timestamp = number.0.into();
```

Comment:
I would use "random" values with more entropy. The `repeat_byte` values are probably ok but this zero is not, as somebody could just set a field to zero accidentally and pass a test because of this.

Reply:
Would a value like `1_700_000_000 + number.0` work? Also, are you suggesting changing existing zero values in tests? This could be a large piece of work, so I'd suggest to do it separately.

Reply:
That's a good idea! I was suggesting you generate a random number and put it there but that's even better.